tools/train/eval_chips.py

```diff
@@ -64,29 +79,39 @@ def train_chips(args):
         f'No log path is given, the default log path will be used: {log_path}')
     logger.info('Initiating the training of the BBAVector model...')
 
-    transform = [torchvision.transforms.RandomHorizontalFlip(), torchvision.transforms.RandomRotation(180),
-                 torchvision.transforms.RandomVerticalFlip()]
+    transform = [torchvision.transforms.RandomVerticalFlip()]
```

Comment:
isn't this supposed to be horizontal flip?

Reply:
Good catch, you're right. The performance increase I've seen when changing that probably stems from the removed random rotation, so the model only had to deal with the incorrect flip. I'll fix that and rerun the experiments, maybe we'll see better results with that.

Reply:
ok. Let me prepare the sets. I will also try to create the chips for you.